geometry_msgs/Point32 target # First evaluation of target position in the map frame
bool is_tower               # Flag to indicate if represents a tower
int8 tower_index             # Index of the tower represented